lib/phoenix_live_reload/application.ex

```diff
@@ -10,11 +10,24 @@ defmodule Phoenix.LiveReloader.Application do
   end
 
   def start_link do
-    opts = [dirs: [Path.absname("")], name: :phoenix_live_reload_file_monitor]
+    opts = [name: :phoenix_live_reload_file_monitor]
```

Comment:
Hi @thehunmonkgroup. I think you can rely on defaults a little more. Here is what I would suggest:

```elixir
dirs = Application.get_env(:phoenix_live_reload, :dirs, [""])
backend_opts = Application.get_env(:phoenix_live_reload, :backend_opts, [])

opts = [
  name: :phoenix_live_reload_file_monitor,
  dirs: Enum.map(dirs, &Path.absname/1),
  backend_opts: backend_opts
]
```

And everything else would stay the same. That should achieve the changes you propose. WDYT?

Reply:
I think it's clear who's been programming Elixir longer... :P

Much cleaner, but doesn't quite do it, because of the way FileSystem passes options to its backends: https://github.com/falood/file_system/blob/v0.2.6/lib/file_system.ex#L5-L18

The ```:backend_opts``` option is invented as a way to allow users of this module to pass additional arguments to FileSystem w/o breaking backwards compatibility. Maybe we should go with a more generic name, ```other_opts```?

See my latest commit, it incorporates your cleaner code with the proper merging of the additional options. Ran some quick tests of the following scenarios, all seem to work well:
 * No ```:phoenix_live_reload``` config
 * Config w/ just ```:backend```
 * Config w/ all opts